cli/install/template.go

```diff
@@ -176,10 +172,8 @@ spec:
         imagePullPolicy: {{.ImagePullPolicy}}
         args:
         - "proxy-api"
-        - "-addr=:8086"
-        - "-metrics-addr=:9996"
-        - "-destination-addr=:8089"
-        - "-telemetry-addr=:8087"
+        - "-destination-addr=127.0.0.1:8089"
+        - "-telemetry-addr=127.0.0.1:8087"
```

Comment:
Would you mind also removing these client addr flags as well, since these match the default?